package.json

```diff
@@ -5,15 +5,15 @@
   "type": "module",
   "main": "./dist/bundle.cjs",
   "module": "./dist/bundle.mjs",
+  "types": "./dist/types/index.d.ts",
   "exports": {
     "import": "./dist/bundle.mjs",
-    "require": "./dist/bundle.cjs",
-    "types": "./dist/index.d.ts"
+    "require": "./dist/bundle.cjs"
   },
-  "types": "./dist/index.d.ts",
   "scripts": {
-    "build": "rm -r dist && tsc && rollup --config",
-    "test": "jest --silent=false && npm run build && npm run typetest",
+    "build": "rm -rf ./dist && tsc --project tsconfig.build.json && rollup --config",
```

Comment:
- When there is no `dist` directory, `rm` throws error
- I missed the proper configuration for `tsc`